vunit/test/unit/test_vhdl_standard.py

```diff
@@ -0,0 +1,65 @@
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Copyright (c) 2014-2019, Lars Asplund lars.anders.asplund@gmail.com
+
+"""
+Test the vhdl_standard.py file
+"""
+
+from vunit.vhdl_standard import VHDL
+
+
+def test_valid_standards():
+    for std in ["93", "02", "08", "19", "1993", "2002", "2008", "2019"]:
+        VHDL.standard(std)
+
+
+def test_error_on_invalid_standard():
+    _assert_is_invalid("2001")
+    _assert_is_invalid("002")
+    _assert_is_invalid("993")
+    _assert_is_invalid("2")
+    _assert_is_invalid("3")
+
+
+def test_equality():
+    assert VHDL.standard("2008") == VHDL.standard("2008")
```

Comment:
Maybe check `VHDL.standard("2008") == VHDL.standard("08")` instead?

Reply:
There is such a check for 1993.

Reply:
Yes. What I meant is that `x == x` will always be true, independently of what `x` is. The only possibility I can imagine for it to fails is if calls `VHDL.standard` where sensitive to some runtime param (system time, number of times the func was executec, etc.). Is this what we are trying to check here?

Reply:
X1 == X2 for a class without __eq__ python compares the object instance id returned by id(X1) and id(X2) which is not equal in the case you commented since the rhs and lhs are different object instances even though they got the same __init__ argument.